src/ripple/protocol/impl/Feature.cpp

```diff
@@ -18,57 +18,73 @@
 //==============================================================================
 
 #include <BeastConfig.h>
+#include <ripple/protocol/Feature.h>
 #include <ripple/basics/contract.h>
 #include <ripple/protocol/digest.h>
-#include <ripple/protocol/Feature.h>
```

Comment:
Putting that include first is a John Lakos-ism.  From "Large Scale C++ Design" page 110, "The `.c` file of every component should include its own `.h` file as the first substantive line of code."  This helps to assure that the `.h` file parses by itself.